db/db_impl.cc

```diff
@@ -383,11 +396,13 @@ Status DBImpl::CloseImpl() {
   ROCKS_LOG_INFO(immutable_db_options_.info_log, "Shutdown complete");
   LogFlush(immutable_db_options_.info_log);
 
-  Status s = Status::OK();
   if (immutable_db_options_.info_log && own_info_log_) {
     s = immutable_db_options_.info_log->Close();
```

Comment:
I think it make it easier to read if we make "Status s = Status::OK();" local to this if clause and then merge it with ret before exiting the clause?

Reply:
Yes to this and previous 2 comments.